main.js

```diff
@@ -0,0 +1,18 @@
+var tableElement = document.querySelector('.table')
+var headerElement = document.querySelector('.header')
+var formElement = document.querySelector('form')
+var noGradesElement = document.querySelector('.no-grades')
+var updateElement = document.querySelector('.update-form')
```

Comment:
Since there's now more than one `<form>` on the page, you should probably use a more specific selector for `formElement`.